src/main/java/anab/Person/ManagePerson.java

```diff
@@ -53,21 +49,25 @@ public static void main(String[] args) {
             System.out.println("it's just an exception named NullPointerException ");
         }
         System.out.println("Average is= " + student.average());
-
+        System.out.println("--------------------------------------------------------------------------------");
         Map<String, Integer> feedbackMarks = new HashMap<String, Integer>();
-        feedbackMarks.put("Matematica", 4);
-        feedbackMarks.put("Romana", 9);
-        feedbackMarks.put("Informatica", 6);
 
-        Trainer trainer = new Trainer("Silvia", "Pompilievici", "Female", 45, "silvia.tr@yahoo.com", false, "Algebra", 4, feedbackMarks);
-        System.out.println("Lista de feedback: " + feedbackMarks);
-        trainer.setExperience("Matematica", 3);
-        trainer.setExperience("Romana", 5);
-        trainer.setExperience("Informatica", 6);
-      /*  trainer.addExperience(4);
-        trainer.getSpecialization();
-        trainer.setSpecialization("Informatica");
+        Trainer trainer = new Trainer("Silvia ", "Pompilievici ", "Female", 45, "silvia.tr@yahoo.com", false, "Algebra", 4, feedbackMarks);
+        System.out.println(trainer.getFirstName() + trainer.getLastName() + "trainer " + trainer.getSpecialization() + " - experienta: " + trainer.getExperience() + " ani");
+        trainer.setSpecialization("Matematica");
+
+        trainer.getExperience();
```

Comment:
since you print the result of this getter at line 60, this call not necessary

Reply:
<trainer.getExperience();>deleted